client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -76,8 +76,8 @@ export const CommentDetailActions = ( {
 					<Gridicon icon="spam" />
 					<span>{
 						isSpam
-							? translate( 'Spam' )
-							: translate( 'Mark as Spam' )
+							? translate( 'Spammed' )
```

Comment:
This sound very odd to me, I've always seen this used as an active verb, like "I've spammed (sent spam email) all the contacts in my address book".
Not that it really matters, since this label shouldn't ever be visible (there is no spam icon/action in the spam list).